recipes/reveal/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "reveal" %}
-{% set version = "0.1" %}
-{% set md5 = "dd45d05f5caabcabb13575e538410c14" %}
+{% set version = "a7b681a" %}
```

Comment:
It is not recommended to use commit ids as version number as conda will not know which one is the highest. The version is still `0.1` in `setup.py` so it would be better to keep the version and just bump the build number. The best would be if we could use a tagged release.